python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -443,15 +464,51 @@ def latest_checkpoint_path(self) -> Optional[Path]:
         else:
             return None
 
+    def _restart(self):
+        self.worker_group.shutdown()
+        if hasattr(self, "_initialization_hook"):
+            initialization_hook = self._initialization_hook
+        else:
+            initialization_hook = None
```

Comment:
Would it make sense to just default `self._initialization_hook` to `None`?

Reply:
Done